client/client.go

```diff
@@ -592,11 +592,16 @@ func Dial(addr string) (c *Client, err error) {
 	return
 }
 
+type Dialer interface {
+	// Dial connects to the given address via the proxy.
+	Dial(network, addr string) (c net.Conn, err error)
```

Comment:
We can remove the name of the return values here, they don't provide any documentation benefits.

Reply:
Done!